api/src/pcapi/core/token/__init__.py

```diff
@@ -20,10 +22,11 @@ class TokenType(enum.Enum):
     EMAIL_CHANGE_CONFIRMATION = "update_email_confirmation"
     EMAIL_CHANGE_VALIDATION = "update_email_validation"
     EMAIL_VALIDATION = "email_validation"
+    PHONE_VALIDATION = "phone_validation"
 
 
 @dataclasses.dataclass(frozen=True)
-class Token:
+class AbstractToken:
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Dans la méthode parente c'est mieux de préciser les fonctions attendues je trouve

```python
class AbstractToken:
    def load_without_checking(self):
        raise NotImplementedError()
```